pkg/dumper/generic/sql.go

```diff
@@ -77,12 +77,12 @@ func (p *sqlDumper) readAndDumpTables(done chan<- struct{}, configTables config.
 		}
 	}
 
+	wg := sync.WaitGroup{}
+	wg.Add(len(configTables.FilterRelashionships(tables)))
```

Comment:
will this change not cause a issue since we look over tables later and do [wg.Done](https://github.com/hellofresh/klepto/pull/77/files#diff-2b065fd5b4d22ac0f646be29c686b35aR107) there?